chapters/operatorsandexpressions.tex

```diff
@@ -1292,6 +1292,13 @@ \subsection{Constant Expressions}\doublelabel{constant-expressions}
   \lstinline!edge!, \lstinline!change!, \lstinline!sample!, and \lstinline!pre!, a function or operator with constant
   subexpressions as argument (and no parameters defined in the function)
   is a constant expression.
+\item
+  Some function calls are constant expressions regardless of the arguments:
```

Comment:
As far as I can see, there's just a single item in the list, which could be reflected when presenting the list:
```suggestion
  There is one function call which is a constant expression regardless of the arguments:
```

Reply:
My logic was that even if ndims is the only function, there are multiple possible calls of it and thus the plural makes sense - and it will be easier if we find other functions that should be in the list.

Reply:
OK, makes sense.